content/support-training.html

```diff
@@ -0,0 +1,171 @@
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.credativ.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/credativ.png" alt="Credativ">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://grafana.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/grafana-labs.svg" alt="Grafana Labs">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://juliusv.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            Julius Volz <br/> 
+            (Independent Contractor)
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://tina.tincho.org" class="btn btn-training btn-default btn-large" target="_blank">
+            Martina Ferrari <br/> 
+            (Independent Contractor)
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.metricfire.com/hosted-prometheus" class="btn btn-training btn-default darkgray-bg btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/metricfire.png" alt="MetricFire">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/robust-perception.png" alt="Robust Perception">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://sysdig.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/sysdig.svg" alt="Sysdig">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.weave.works/features/prometheus-monitoring/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/weaveworks.svg" alt="Weaveworks">
+          </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content">
+        <a name="add"></a><h2>Add a resource</h2>
+        <p>
+          Anyone who has developed a Prometheus training program or offers related services
+          can add themselves to this page if they meet the following requirements.
+        </p>
+        </div>
+        
+        <div class="col-md-6 doc-content">
+          <!-- NOTE: WIP -->
```

Comment:
I'd just not mention this, the standard is currently applied is that you send a PR.

Reply:
Why would we not give some expectations for those PRs so they arrive in better shape before the first review round?

Reply:
We don't ask for e.g. linkedin pages though, this is proposing new requirements that I don't see a need for.

Reply:
I think those requirements are pretty reasonable to add / include though. It doesn't have to be specifically LinkedIn of course, but that's also not how I read the wording.

Reply:
> If the course is instructor led, you must provide credentials (a link to your LinkedIn profile or resume will suffice)

that is strongly suggested

Reply:
We could make the language even clearer by prepending "for example, a link to...", but I think overall the requirement of linking to some kind of credentials / bio / ... of a consultant or course instructor makes sense.

Reply:
I'm not seeing the point here. Am I meant to check if the word "Prometheus" appears on the profile? When should I reject a PR for the instructor being too inexperienced?

If it's an impartial list, I think we should keep it as an impartial list.

Reply:
The point would be for the visitor of the page to be able to find relevant information about the linked person. They can then decide for themselves if they find that person trustable enough based on the linked info...

Reply:
I don't think that we should be mandating things about what information in presented on a vendor's landing page and in what form, among other things that raises privacy/legal issues. For example you're asking RP to link to the personal LinkedIn of an ex-employee.

Reply:
Again, it doesn't need to be LinkedIn. But asking for some form of background info about an instructor or individual consultant who wants to add themselves to our site makes sense, no?

Reply:
In principle, but the practicalities are problematic. For example I'm not sure if the SaaS that RP uses for our training supports this.

I think this is something that should be at most guidance to vendors on how to have a good landing page, not anything we actually try to enforce.

Reply:
Ok, if it's reworded from a requirement to recommendation, that would also be fine with me.

Reply:
I think this is a case of trying to find a technical solution to a social problem, in effect lawyering a hypothetical issue into a possible blocker.

Prometheus tends to work as a consensus-based project in which there's acceptable uncertainty space to operate in. Listing an example is not mandating a requirement.
If we try to lawyer every corner case, we will not get anything done in sane time frames.
If we prevent any changes, there is no way to build up new operational guidelines.

And: If today we got a PR with a new company being listed, we would do some due diligence anyway. So there is both precedent and implicit consensus for having requirements and not just merging _anything_.

Reply:
As presently written in this PR, these would be new hard requirements.

> So there is both precedent and implicit consensus for having requirements and not just merging anything.

The precedent I've seen is that there is no bar - beyond someone asking to be listed. One of the current entries was added at a time when their website made no mention of that company providing Prometheus services.

If a PR looks weird I'll certainly double check with the author, but I won't reject it.